precompiles/ibc/abi.json

```diff
@@ -0,0 +1,46 @@
+[
```

Comment:
keeping formatted for readability for now (will flatten when code is post-draft)